magento2/magento2/scripts/init-and-run.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -x
+
+# Turn on monitor mode, required by job control
+set -m
+
+if [ ! -f /var/www/html/magento2/.installed ]; then
+  cd /var/www/html/magento2/ || exit
+  composer config --global repos.packagist composer https://packagist.org
+  composer require nosto/php-sdk:@stable
+
+  until mysql -u"${MYSQL_USER}" -p"${MYSQL_PASSWORD}" -h"${MYSQL_HOST}"; do
```

Comment:
This should be done in the docker-compose using a health-check https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y

Reply:
I couldn't configure that using docker-compose file version 3.
In order to move forward with this PR, there's an open issue here for later improvement:
https://github.com/Nosto/docker-platforms/issues/2